data/assets/scene/solarsystem/planets/jupiter/minor/carpo_group.asset

```diff
@@ -35,15 +35,80 @@ local carpoGroup = {
   }
 }
 
+-- Generate labels for each moon
+local moon_labels = {}
+
+for i, moon in ipairs(carpoGroup) do
+  moon_labels[i] = {
+    Identifier = moon.Identifier .. "Label",
+    Parent = moon.Identifier,
+    Renderable = {
+      Enabled = false,
+      Type = "RenderableLabel",
+      Text = moon.Identifier,
+      FontSize = 70.0,
+      Size = 7.1,
+      MinMaxSize = { 1, 25 },
+      OrientationOption = "Camera View Direction",
+      BlendMode = "Additive",
+      EnableFading = true,
+      FadeUnit = "Gm",
+      FadeDistances = { 1.0, 100.0 },
+      FadeWidths = { 0.0, 100.0 }
+    },
+    Tag = { "solarsystem_labels", "moon_labels", "minor_moon_labels" },
+    GUI = {
+      Name = moon.Identifier .. " Label",
+      Path = "/Solar System/Planets/Jupiter/Moons",
+      Description = "Label for Jupiter's moon " .. moon.Identifier .. " (Carpo group)"
+    }
+  }
+
+  -- Hacky solution to issue where the main label gets culled due to it's onscreen size.
```

Comment:
-> its

Reply:
Fixed in: [055eab06540015eba3138c73cb4cb4bf6f36b5ad](https://github.com/OpenSpace/OpenSpace/commit/055eab06540015eba3138c73cb4cb4bf6f36b5ad)